File: unit11-hw/ruby/Owl.rb
# Owl.rb | Defines the Owl class which inherits attributes and methods
#   from the Bird superclass.
#
# Begun by: Dr. Jump for CS 315 at King's College.
# Completed by: Robert Lowman
# Date: 11/29/16
####################################################

require_relative 'FlyingBird.rb'

class Owl < FlyingBird

  def call
    'Whoo-hoo'
  end

end

File: unit11-hw/python/Owl.py
# Owl.py | Defines the Owl class which inherits attributes and methods
#   from the Bird superclass.
#
# Completed by: Robert Lowman
# Date: 12/7/16
####################################################
import FlyingBird

class Owl(FlyingBird.FlyingBird):

    def __init__(self, name):
        FlyingBird.__init__(name)
        self.name = name

    def call(self):
        return 'Hoot!'

    def className(self):
        return 'Owl'
